static/app/gettingStartedDocs/go/martini.tsx

```diff
@@ -11,9 +11,7 @@ import {t, tct} from 'sentry/locale';
 // Configuration Start
 export const steps = ({
   dsn,
-}: {
-  dsn?: string;
-} = {}): LayoutProps['steps'] => [
+}: Partial<Pick<ModuleProps, 'dsn'>> = {}): LayoutProps['steps'] => [
```

Comment:
Should this be extracted into a `StepsProps` type or something?

Reply:
I don't think so, because the DSN is used in the content of step prop and we can not inject it out of the box as the `Sentry.init` is different for each platform. Users updating the Doc have to decided where to add it to the code